### PR TITLE
Create 2021-11-22-IT-curriculum-update

### PR DESCRIPTION
**Pull request by karenword**
This still needs a header and review before publication. Also, I think there were some problems with twitter HTML in the last post that may apply here? 

Feedback is very welcome; I will update the PR status when I feel more confident that this is ready for final review & publication.

### CODE REVIEW

**Comment by karenword** [comment]
Also still needs a link to the curriculum release in the first paragraph after @fmichonneau has updated authors and editors.

File: _posts/2021/11/2021-11-18-IT-curriculum-update.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+authors: ["Karen Word"]
+teaser: "The new iteration of the Instructor Training curriculum has been submitted for release"
```
Comment (klbarnes20) [inline _posts/2021/11/2021-11-18-IT-curriculum-update.md]:
```suggestion
teaser: "The new version of the Instructor Training curriculum has been released"

Just trying to make this a little catchier 
```

File: _posts/2021/11/2021-11-18-IT-curriculum-update.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+title: "The Carpentries Instructor Training Curriculum has been Updated!"
+date: 2021-11-18
+time: "09:00:00"
+tags: ["Instructor Trainers"]
+---
+
+
+Throughout most of 2021, The Carpentries Core Team and Trainer community have been developing, discussing, and testing an updated version of our Instructor Training curriculum. It has now been merged into our repository on GitHub and submitted to Zenodo for official release!
```
Comment (klbarnes20) [inline _posts/2021/11/2021-11-18-IT-curriculum-update.md]:
```suggestion
Throughout 2021, The Carpentries Core Team and Trainer community have been developing, discussing, and testing an updated version of our Instructor Training curriculum. It has now been merged into our repository on GitHub and submitted to Zenodo for official release!
```

File: _posts/2021/11/2021-11-18-IT-curriculum-update.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+### Changes in this update include:
+* General changes to make the curriculum more useable and maintainable:
+    - [ ] Reduced hyperlinks to external websites 
+    - [ ] Eliminated use of handouts for activities
+    - [ ] Reduced duplication of policy articulated elsewhere; increased orientation to resources
```
Comment (klbarnes20) [inline _posts/2021/11/2021-11-18-IT-curriculum-update.md]: I don't feel like this line is clear to people who haven't been part of our conversations.

File: _posts/2021/11/2021-11-18-IT-curriculum-update.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+* Content removed or transferred to [optional exercises page](https://carpentries.github.io/instructor-training/additional_exercises/index.html) (more relevant to lesson development, which is now a separate curriculum)
+    - [ ] Practice writing multiple choice questions
+    - [ ] Practice writing learner profiles
+    - [ ] Content on Bloom's Taxonomy
+* Content changed or rearranged
+    - [ ] Previous episode on "Mindset" reduced and combined with ["Motivation and Demotivation" episode](https://carpentries.github.io/instructor-training/08-motivation/index.html)
+    - [ ] Previous content on Equity, Inclusion, and Accessibility in "Motivation and Demotivation" episode moved to [its own episode](https://carpentries.github.io/instructor-training/09-eia/index.html) and expanded
+    - [ ] Previous references to formative assessment sprinkled across episodes are now highlighted as distinct uses
+    - [ ] Mental models are now [explained using concept mapping](https://carpentries.github.io/instructor-training/02-practice-learning/index.html) with addition of analogies to summarize a recognizable model
```
Comment (klbarnes20) [inline _posts/2021/11/2021-11-18-IT-curriculum-update.md]: I find the phrase "analogies to summarize a recognizable model" a little unclear

File: _posts/2021/11/2021-11-18-IT-curriculum-update.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+This workflow was not entirely satisfactory. One effect was to narrow the audience for early conversations, which occurred in the "quieter" data-lessons repository and was mostly restricted to Maintainers and a few Trainers who were paying close attention. Some found this process to be too secretive, preferring that all conversation remain in our main repository. In addition, the many ways to provide feedback (issues or comments on the PR in the main repository; issues or comments on PRs in data-lessons) proved confusing for both contributors and maintainers.
```
Comment (klbarnes20) [inline _posts/2021/11/2021-11-18-IT-curriculum-update.md]: secretive doesn't feel like the right word. Maybe isolated? Maybe separated from our normal discussions? Maybe inaccessible?

**Review by klbarnes20 — APPROVED** [review]
Minor comments, but this looks good to me!

File: _posts/2021/11/2021-11-18-IT-curriculum-update.md (hunk 3 above)
Comment (karenword) [reply _posts/2021/11/2021-11-18-IT-curriculum-update.md]:
```suggestion
    - [ ] Reduced content on Carpentries policies; increased orientation to main sources of policy information
```

File: _posts/2021/11/2021-11-18-IT-curriculum-update.md (hunk 4 above)
Comment (karenword) [reply _posts/2021/11/2021-11-18-IT-curriculum-update.md]:
```suggestion
    - [ ] Mental models are now [explained using concept mapping](https://carpentries.github.io/instructor-training/02-practice-learning/index.html) and analogies
```

File: _posts/2021/11/2021-11-18-IT-curriculum-update.md (hunk 5 above)
Comment (karenword) [reply _posts/2021/11/2021-11-18-IT-curriculum-update.md]:
```suggestion
This workflow was not entirely satisfactory. One effect was to narrow the audience for early conversations, which occurred in the "quieter" data-lessons repository and was mostly restricted to Maintainers and a few Trainers who were paying close attention. Some found this process to be too isolated, preferring that all conversation remain in our main repository. In addition, the many ways to provide feedback (issues or comments on the PR in the main repository; issues or comments on PRs in data-lessons) proved confusing for both contributors and maintainers. 
```

**Comment by karenword** [comment]
This is now ready to publish as soon as the Zenodo link is added (at least, from my end!).

**Pull request merged**